### DISCUSSION TITLE
insult to the nation

### DISCUSSION THREAD

**Discussion by paskalsq** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#83] [General]
Putin, smarter than you, it's useless to discuss politics

**Comment by HerMajestyDrMona** [comment]
If it's useless to discuss politics, why you're creating a political discussion here? You seem to be as smart as your Putin, which rather brings a negative conclusion.

**Reply by Pablo-Diablo** [reply]
Biden is a fascist. You follow him blindly which makes you a fascist by proxy. All fascists will be destroyed whether you like it or not.

**Reply by HerMajestyDrMona** [reply]
Where the hell do you see anybody mentioning Biden? It feels pity to see that in 2023, people are still not getting schizophrenia treatment :(

**Reply by RijonBD** [reply]
**Benjamin Netansatyanhuuu is a war criminal**

**Reply by HerMajestyDrMona** [reply]
**Hamas and Hezbollah are terrorist organizations.**
